src/pages/signup.vue

```diff
@@ -63,22 +63,16 @@ export default {
         }
     },
     methods:{
-        async submitForm(){
-            this.$Auth().createUserWithEmailAndPassword(this.email,this.password).then(data =>{
-                data.user.updateProfile({
-                    displayName:this.firstname+this.lastname
-                }).then(()=>{ 
-                    alert("created login form successfully:"+this.email);
-
-                }).catch((error)=>{
-                    console.log("error",error);
-                    
-                })
-                    
-            
-            })
-
+      ...mapActions(["signUp"]),
+      submitForm(){
```

Comment:
@pavankumarhb always add the comments on what this method does for eg: 
// Submit handler for user signup.